apisix/core/ctx.lua

```diff
@@ -159,10 +160,22 @@ do
                     val, err = cookie:get(sub_str(key, 8))
                     if err then
                         log.warn("failed to fetch cookie value by key: ",
-                                 key, " error: ", err)
+                                key, " error: ", err)
                     end
                 end
 
+            elseif core_str.has_prefix(key, "arg_") then
+                local args = t.args
+                local arg_key = sub_str(key, 5)
+                local m, err = ngx_re_match(args, "(^|&)" .. arg_key .. "=([^&]*)(&|$)", "jo")
```

Comment:
We can use `core.request.get_uri_args`, which is more clear?

Reply:
Do you mean?
```
args :　a-b=ab
core.request.get_uri_args_a-b :  ab
```

if  *uri*  is for making args different from args in nginx var，  I think the following example is clear enough?
```
args :　a-b=ab
core.request.get_args_a-b :  ab
```

Reply:
Something like `core.request.get_uri_args()["a-b"]`, note that if the result is a table, get the first one.

Reply:
```
                        {
                            "vars": [
                                ["uri_args_a-b","==","ab"]
                            ]
                        }
```
If get_uri_args return a table,  wil this work properly?

Reply:
Note that if the result is a table, get the first one.

Reply:
get it